modules/mapping-format-detector/src/main/java/cn/maxpixel/mcdecompiler/mapping/detector/DefaultDetectionUnit.java

```diff
@@ -26,8 +27,9 @@ public boolean canDetectHeader() {
         String s = contents.get(0);
         if (s.startsWith("PK: ") || s.startsWith("CL: ") || s.startsWith("FD: ") || s.startsWith("MD: ")) return Pair.of(MappingFormats.SRG, Percentage.NINETY_NINE);
         else if (s.endsWith(":")) return Pair.of(MappingFormats.PROGUARD, Percentage.NINETY);
+        else if (s.contains("¶") || s.contains("\\u00B6")) return Pair.of(MappingFormats.PDME, Percentage.NINETY);
```

Comment:
you should only choose one, either header detection or content detection

Reply:
ok i chose content